aptos-move/framework/aptos-stdlib/sources/hash.move

```diff
@@ -65,6 +65,17 @@ module aptos_std::aptos_hash {
         ripemd160_internal(bytes)
     }
 
+    /// Returns the BLAKE2B-256 hash of `bytes`.
+    ///
+    /// WARNING: TODO
```

Comment:
Does this need to be fleshed out?

Reply:
I followed the template from other hashes. Wasn't sure what to put there. Happy to get suggestions

Reply:
No warning is necessary here.

Reply:
Just remove and mark as resolved.